### ISSUE TITLE
Any way to run widget remotely?

### ISSUE THREAD

**Issue by Boohooligan** [3dproger/AxelChatWidgets#12]
Hi, I have kind of an unusual use case, but I was trying to figure out if it was possible to run the messages widget from another device on my local network. I opened 8355 and 8356 on the machine running AxelChat and copied the widgets folder to the other machine, updated the websocket connections in the javascript that I could find to route to the other device by IP instead of localhost, but I couldn't get it to work.

Basically, I am looking to cast AxelChat from one device to another. It seemed like the widget would be the best way to do it, but also open to other possibilities.

**Comment by 3dproger** [comment]
Hi!

This feature is already planned, but it is difficult to say when the results will be

**Comment by 3dproger** [comment]
Hi! If it is relevant, here is some information that will be useful: https://github.com/3dproger/AxelChat/discussions/597
